### PR TITLE
list converted to json

### PR DESCRIPTION
**Pull request by XiaoPa**
list converted to json, I made things complex. now is fine

### CODE REVIEW

File: src/flask_app.py [hunk 1]
```diff
@@ -53,4 +53,4 @@ def server_error(exception):
 	handler.setFormatter(formatter)
 	app.logger.info('data sorting services is running')
 	app.logger.addHandler(handler)
-	app.run('0.0.0.0', 5000)
+	app.run()
```
Comment (maZahaca) [inline src/flask_app.py]: It doesn't work without that `'0.0.0.0', 5000`

**Review by maZahaca — CHANGES_REQUESTED** [review]

File: src/flask_app.py (hunk 1 above)
Comment (maZahaca) [reply src/flask_app.py]: Do you have problems to run it locally with it?

Comment (XiaoPa) [reply src/flask_app.py]: sorry, I forgot to add `'0.0.0.0',5000` back, I have to remove it to be able to test locally.

**Comment by maZahaca** [comment]
Good job!

**Pull request merged**